torchci/rockset/commons/__sql/commit_failed_jobs.sql

```diff
@@ -2,19 +2,20 @@
 -- used to decide if a failure is due to broken trunk
 SELECT
   j.id,
-  j.name as jobName,
-  CONCAT(w.name, ' / ', j.name) as name,
+  j.name AS jobName,
+  CONCAT(w.name, ' / ', j.name) AS name,
+  j.runner_name AS runnerName,
   j.conclusion,
   j.completed_at,
   j.html_url,
   j.head_sha,
   j.head_branch,
   j.torchci_classification.captures AS failure_captures,
   j.torchci_classification.line AS failure_line,
-  j._event_time as time,
+  j._event_time AS time,
```

Comment:
I'll follow up with a separate PR to remove the usage of `_event_time` in these queries